README.md

```diff
@@ -28,9 +28,9 @@ See the [documentation](docs/README.md) for more information.
 
 # Development
 
-Checkout the repository and get a development environment with `script/setup`. This will create a new virtual environment in the `venv` directory of the repository, and install all necessary requirements.
+Checkout the repository and get a development environment with `script/setup`. This will create a new virtual environment in the `venv` directory of the repository, and install all necessary requirements. It will also build your code space to install latest dependencies, so you should run it again occasionally.
 
-Before developing, always activate your virtual environment with `source venv/bin/activate`.
+Before developing, always activate your virtual environment with `source venv/bin/activate`. This handles python dependencies so that they do not conflict with other python projects. You can deactivate the virtual environment by running `deactivate`.
```

Comment:
Explaining what a virtual environment is seems out of scope for the README. We better just link the term in the previous paragraph to the official documentation.

Reply:
If one is not running this on some codespaces or cloud, you're practically asking people to just start a virtual environment on their own computer and recommending them to run it always, but not explaining even in two sentences what it is or how they can "undo" the activation to get to back to the old state.

Adding two sentences to ease users mind did not seem such a big thing, but I'm fine either way. I'm just trying to help others. 😃 

Shall I close this PR?